src/index.css

```diff
@@ -1,13 +1,37 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family: "SangBleu OG Serif Light Regular";
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  text-decoration: none;
+}
+
+p {
+  margin: 0;
+  font-family: "SangBleu OG Serif Light Regular";
```

Comment:
you dont need these `font-family` rules in here (and in a and maybe button) since they are already in `body`

Reply:
Removed. I've just had to leave it in for `button`, it's the only one that didn't inherit from the `body`.